addons/crm/static/src/views/fields/crm_statusbar_field.js

```diff
@@ -2,14 +2,8 @@
 
```

Comment:
This whole widget is probably not necessary anymore as the code in selectItem is the same